packages/council-proposals/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@elementfi/council-proposals",
+  "version": "1.0.2",
+  "description": "",
+  "dependencies": {
+    "@elementfi/elf-council-tokenlist": "^1.0.2",
+    "@elementfi/elf-council-typechain": "^1.0.2",
+    "@types/lodash.uniq": "^4.5.6",
+    "@types/lodash.zip": "^4.2.6",
+    "graphql": "^16.3.0",
+    "graphql-request": "^4.0.0",
+    "lodash.uniq": "^4.5.0",
+    "lodash.zip": "^4.2.0",
+    "tsc": "^2.0.3",
+    "tsconfig-paths": "^3.10.1"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "types": "dist/types.d.ts",
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-vyper": "^2.0.1",
+    "@openzeppelin/contracts": "^3.4.0",
+    "@typechain/ethers-v5": "^7.0.1",
+    "@typechain/hardhat": "^2.2.0",
+    "@types/ethereumjs-abi": "^0.6.3",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^14.14.22",
+    "@typescript-eslint/eslint-plugin": "^4.28.4",
+    "@typescript-eslint/parser": "^4.28.4",
+    "eslint": "^7.31.0",
+    "eslint-config-prettier": "^8.3.0",
+    "ethers": "^5.4.1",
+    "hardhat": "^2.5.0",
+    "prettier": "^2.3.2",
+    "prettier-plugin-solidity": "^1.0.0-alpha.58",
+    "solhint-plugin-prettier": "^0.0.5",
+    "ts-generator": "^0.1.1",
+    "ts-node": "^10.1.0",
+    "typechain": "^5.1.2",
+    "typechain-target-ethers-v5": "^5.0.1",
+    "typescript": "^4.3.5"
+  },
+  "scripts": {
+    "build-testnet": "bash scripts/build-testnet.sh && npm run _build-index && npm run _copy-files",
+    "build-goerli": "bash scripts/build-goerli.sh && npm run _build-index && npm run _copy-files",
+    "build-mainnet": "bash scripts/build-mainnet.sh",
+    "build": "npm run build-goerli && npm run build-mainnet && npm run build-testnet && npm run _build-index && npm run _copy-files",
+    "_build-index": "bash scripts/build-index.sh",
+    "_copy-files": "bash scripts/copy-dist-files.sh",
+    "_copy-testnet-addresses": "cp ../elf-council-testnet/src/addresses/testnet.addresses.json src/addresses/testnet.addresses.json",
```

Comment:
elf-council-testnet is now council-testnet in the monorepo, `../apps/council-testnet/src/addresses/testnet.addresses.json`